web/frontend/src/components/slam/index.vue

```diff
@@ -253,17 +224,13 @@ const stopMoveOnMap = () => {
 };
 
 const refresh2d = async (name: string) => {
-  const flags = await fetchFeatureFlags(name);
-
   const map = await fetchSLAMMap(name);
   const returnedPose = await fetchSLAMPose(name);
 
-  // TODO: Remove this check when APP and carto are both up to date [RSDK-3166]
-  if (flags && flags.response_in_millimeters) {
-    returnedPose.setX(returnedPose.getX() / 1000);
-    returnedPose.setY(returnedPose.getY() / 1000);
-    returnedPose.setZ(returnedPose.getZ() / 1000);
-  }
+  returnedPose.setX(returnedPose.getX() / 1000);
```

Comment:
While we're here, mind adding a comment that the pose is returned in mm, but map is displayed in meters, so we convert to meters here?